app/src/androidTest/java/com/example/avjindersinghsekhon/minimaltodo/testApp/tests/HomePageValidationTest.java

```diff
@@ -0,0 +1,99 @@
+package com.example.avjindersinghsekhon.minimaltodo.testApp.tests;
+
+import android.support.v7.widget.LinearLayoutCompat;
+import android.widget.ImageButton;
+import android.widget.ImageView;
+
+import com.example.avjindersinghsekhon.minimaltodo.R;
+
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.assertion.PositionAssertions.isAbove;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.hasSibling;
+import static android.support.test.espresso.matcher.ViewMatchers.isClickable;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withParent;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.is;
+
+public class HomePageValidationTest extends EspressoTestBase {
+
+    @Test
+    public void Validation() {
```

Comment:
The name of this test should be more specific. The name should be the answer of the question: "What are we validating here?".